### ISSUE TITLE
Fix 0.20 deprecation message

### ISSUE THREAD

**Issue by kakawait** [kakawait/hugo-tranquilpeak-theme#138]
```
WARNING: Site config's rssURI is deprecated and will be removed in a future release. Set baseName in outputFormats.RSS.
```

**Comment by jhabdas** [comment]
Do nothing at it's already fixed: https://github.com/spf13/hugo/issues/3319

**Comment by kakawait** [comment]
@jhabdas nice for tracking and good to heard :) I will still open until Hugo `0.20.1` will be released to avoid other people open the issue 

Thank you again

**Comment by jhabdas** [comment]
That's awfully kind of you. It reminds me of proverb:

> give a man a fish and you feed him for a day; teach a man to fish and you feed him for a lifetime

**Comment by kakawait** [comment]
Confirmed fixed on `0.20.1` which it was release last day
